testing/integration/setup_test.go

```diff
@@ -110,6 +111,10 @@ func teardownHarness(t *testing.T, harness *appHarness) {
 	if err != nil {
 		t.Errorf("Error closing database context: %+v", err)
 	}
+	err = os.RemoveAll(harness.config.DataDir)
```

Comment:
Didn't we remove this thing some time ago to make sure we do have the datadir left in case of failures? (also in leveldb_test)

Reply:
Removed. I do think we should delete the datadir if nothing fails, but whatever. (we already delete it for the rest of the tests)